likecoin/js/admin-metabox/metabox.js

```diff
@@ -283,11 +283,13 @@ async function onISCNCallback(data) {
 async function onSubmitToISCN(e) {
   if (e) e.preventDefault();
   const { siteurl } = wpApiSettings;
+  const { url } = lcPostInfo;
   lcPostInfo.mainStatus = 'onRegisterISCN';
   updateFieldStatusText(ISCNStatusTextField, getStatusText(lcPostInfo.mainStatus));
   const redirectString = encodeURIComponent(siteurl);
   const iscnId = encodeURIComponent(lcPostInfo.iscnId || '');
-  const likeCoISCNWidget = `${ISCN_WIDGET_ORIGIN}/in/widget/iscn-ar?opener=1&blocking=1&platform=wordpress&redirect_uri=${redirectString}&iscn_id=${iscnId}`;
+  const urlString = encodeURIComponent(url);
+  const likeCoISCNWidget = `${ISCN_WIDGET_ORIGIN}/nft/url?opener=1&platform=wordpress&redirect_uri=${redirectString}&url=${urlString}&iscn_id=${iscnId}&update=${iscnId ? 1 : 0}`;
```

Comment:
the name of the const is ambiguous after we use app.like.co as widget 
```suggestion
  const iscnWidgetURL = `${ISCN_WIDGET_ORIGIN}/nft/url?opener=1&platform=wordpress&redirect_uri=${redirectString}&url=${urlString}&iscn_id=${iscnId}&update=${iscnId ? 1 : 0}`;
```